src/components/header.svelte

```diff
@@ -0,0 +1,21 @@
+<div class="header">
+	<img class = "fit" src = "https://writingstudiestree.org/live/sites/default/files/WST_Logo_Green.jpg" alt = "scholarly books on a shelf"/>
+</div>

```

Comment:
Was this our old alt text? Yeesh, kind of a misrepresentation. Better would be: "the letters W, S, T over a stylized network diagram, in green."